exp/query/grpc.go

```diff
@@ -35,5 +40,31 @@ func NewGRPC(api *API, log logrus.FieldLogger) *GRPC {
 }
 
 func (g *GRPC) Search(ctx context.Context, req *protocol.SearchRequest) (*protocol.SearchReply, error) {
+	// TODO only create class if not exists/not changed? replace with go client and do not hardcode the url
```

Comment:
IMO if we search a class that doesn't exists then we shouldn't create it. It should be created and the shard should be in S3 already, so the work we have to do is querying the schema to understand which class+shard exists and we can pull.

Reply:
100% agree, My understanding is also auto-schema and auto-tenant creation should be on the import path, not on the query path. Unless I miss something.

Reply:
either Nate or myself will handle this case in the follow up PR. Resolving conversation to unblock the merge.